test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -91,10 +91,11 @@ class CloudFoundryDeployTest extends BasePiperTest {
         nullScript.commonPipelineEnvironment.configuration = [
             general: [
                 camSystemRole: 'testRole',
-                cfCredentialsId: 'myCreds'
+                cfCredentialsId: 'test_cfCredentialsId'
             ],
             stages: [
                 acceptance: [
+                    cfAppName: 'testAppName',
```

Comment:
Why do we need to change this?

Reply:
Actually this change is left over from a previous version of the test, when I did not set the `buildTool` to MTA, since in case of the `cf_native` `deployTool` there will be the attempt to read the `cfAppName` from the `manifest.yml`.